mtp_noms_ops/apps/security/templatetags/security.py

```diff
@@ -82,6 +83,22 @@ def format_resolution(resolution):
     return credit_resolutions.get(resolution, resolution)
 
 
+@register.filter
+def check_description(check):
+    description = check['description']
+    if isinstance(description, list):
+        description = '. '.join(description)
+    if description == 'Credit matched no rules and was automatically accepted':
```

Comment:
I'm a bit wary of matching on magic strings. Besides why would we want to exclude this?

Reply:
it's a fallback in case this filter is accidentally used for a security check that was auto-accepted, i.e. one which should _not_ be displayed using this filter

Reply:
Hmm, i'm not entirely convinced that it's worth adding this complexity if we have confidence in our data quality. Maybe add a `logger.warning(` and we can remove this code if nothing trips it after a couple of weeks? WDYT?

Reply:
as i say, i'm ok with removing it – is that what you'd like? i don't think we'd ever notice a log message – at present there is no such place that a log could be emitted so if we did put it in, we'd have to leave it forever. i don't think that's worth it. shall i remove this part?